contracts/ForkDelta.sol

```diff
@@ -176,35 +124,45 @@ contract ForkDelta is SafeMath {
     uint feeMakeXfer = 0;
     uint feeTakeXfer = 0;
     if (now >= freeUntilDate) {
-      feeMakeXfer = safeMul(amount, feeMake) / (1 ether);
-      feeTakeXfer = safeMul(amount, feeTake) / (1 ether);
+      feeMakeXfer = amount.mul(feeMake) / (1 ether);
```

Comment:
Do we need a SafeMath division?

Reply:
No. Dividing by 1 ether will never result in an overflow.